From 3eabb76b451f8b89e98975ada9b9369c7e8e4bc3 Mon Sep 17 00:00:00 2001
From: Pavan S <psreenivas@habana.ai>
Date: Mon, 20 Feb 2023 11:41:07 +0200
Subject: [PATCH] [SW-123737] gaudi2: ext_rom: fix bug in software all reset

The software all reset was not working because the reset
file was not getting compiled for erom recovery image
and the weak definition of hl_dev_reset was getting
executed. This patch fixes this by including the reset
file for erom recovery image.

Signed-off-by: Pavan S <psreenivas@habana.ai>
Change-Id: Ifa220ae48f871597e6c7c8febe1c061b3b0411d0
---

diff --git a/app/ext_rom/Makefile b/app/ext_rom/Makefile
index 2a2335f..ac46f28 100644
--- a/app/ext_rom/Makefile
+++ b/app/ext_rom/Makefile
@@ -102,6 +102,7 @@
 	$(HL_DEV_ARCH)/../common/pll.o \
 	$(HL_DEV_ARCH)/../common/sha.o \
 	$(HL_DEV_ARCH)/../common/$(HL_TARGET).o \
+	$(HL_DEV_ARCH_COMMON)/reset.o \
 	$(HL_DEV_ARCH)/$(HL_TARGET).o
 
 EROM_RECOVERY_EXTRA_LIBS := "-lfdt"
